manager-bundle/src/Command/DebugPluginsCommand.php

```diff
@@ -131,16 +131,16 @@ private function listBundles(): int
             $class = \get_class($bundle);
 
             if (ContaoModuleBundle::class === $class) {
-                $path = sprintf('system/modules/%s', $name);
+                $path = Path::join('system/modules', $name);
             } else {
                 $reflection = new \ReflectionClass($class);
 
-                if (is_dir($dir = \dirname($reflection->getFileName()).'/Resources/contao')) {
-                    $path = (new Filesystem())->makePathRelative($dir, $this->kernel->getProjectDir());
+                if (is_dir($dir = Path::join($reflection->getFileName(), '../Resources/contao'))) {
+                    $path = Path::makeRelative($dir, $this->kernel->getProjectDir());
                 }
             }
 
-            $rows[] = [$bundle->getName(), $path];
+            $rows[] = [$bundle->getName(), $path.'/'];
```

Comment:
Can’t we omit the trailing slash here?

Reply:
see https://github.com/contao/contao/pull/1902#discussion_r450117526